DomainServices/Services/CustomerServices.cs

```diff
@@ -3,67 +3,63 @@
 using System.Collections.Generic;
 using System.Linq;
 
-namespace APIDesafioWarren.DataBase
+namespace APIDesafioWarren.DomainService
 {
     public class CustomerServices : ICustomerServices
     {
-        private readonly List<Customer> _customers = new List<Customer>();
+        private readonly List<Customer> _customersServices = new List<Customer>();
```

Comment:
```suggestion
        private readonly List<Customer> _customerServices = new();
```
Podemos simplificar assim